plugins/PRAplugin/doc/version.tex

```diff
@@ -0,0 +1 @@
+bc7dfe29dfa9b18998f61708f50af2de94fe874e Diego Mandelli Tue, 24 Apr 2018 10:07:59 -0600
```

Comment:
this one is atuomatically genereted right? not need to add

Reply:
removed